sitemap.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Just for the history, the only reason sitemap.xml and style.css are modified here is because the initial new line endings were CRLF but they are now LF, the standard for my local IDE. Something that should be covered by prettier, so I will work on that eventually.